zipline/finance/performance/tracker.py

```diff
@@ -523,12 +517,31 @@ def _calculate_rolling_expected_shortfall(self, data_portal):
         )
 
         asset_returns = zp.asset_returns_for_cvar(
-            assets=assets,
+            assets=list(weights.columns),
             benchmark=self.benchmark_asset,
             data_portal=data_portal,
             end_date=sim_params.end_session,
             lookback_days=len(sim_params.sessions) + days_before_start,
         )
+        asset_returns_values = asset_returns.values
+
+        def rolling_shortfall():
```

Comment:
If we decide to use this, I'd probably break this out into a function that takes `asset_returns`, `weights`, and `cvar_cutoff` as parameters.